modules/Bio/EnsEMBL/Compara/PipeConfig/ncRNAtrees_conf.pm

```diff
@@ -82,6 +83,14 @@ sub default_options {
             'compara_curr',
         ],
 
+     # Whole db DC parameters
+     'datacheck_groups' => ['compara_gene_tree_pipelines'],
+     'db_type'          => ['compara'],
+     'output_dir_path'  => $self->o('work_dir') . '/datachecks/',
+     'overwrite_files'  => 1,
+     'failures_fatal'   => 1, # no DC failure tolerance
+     'db_name'          => $self->o('dbowner') . '_' . $self->o('pipeline_name'),
```

Comment:
The indentation in this subroutine is not entirely consistent, but this change
would keep at least keep these parameters in line with their neighbours.
```suggestion
        # Whole db DC parameters
        'datacheck_groups' => ['compara_gene_tree_pipelines'],
        'db_type'          => ['compara'],
        'output_dir_path'  => $self->o('work_dir') . '/datachecks/',
        'overwrite_files'  => 1,
        'failures_fatal'   => 1, # no DC failure tolerance
        'db_name'          => $self->o('dbowner') . '_' . $self->o('pipeline_name'),
```